.github/actions/update-documentation/action.yml

```
@@ -0,0 +1,86 @@
+name: "Deploy Keptn Lifecycle Toolkit on GH cluster"
+description: "Creates a Kind cluster and deploys Keptn Lifecycle Toolkit"
+inputs:
+   version:
+     required: true
+     description: "Version of the Keptn Lifecycle Toolkit Documentation to be deployed"
+   doc-repo:
+     required: true
+     description: "Path to the documentation repository"
+     default: "keptn-sandbox/lifecycle-toolkit-docs"
+   examples-repo:
+     required: true
+     description: "Path to the documentation repository"
+     default: "keptn-sandbox/lifecycle-toolkit-examples"
+   klt-repo:
+     required: true
+     description: "Path to the klt repository"
+     default: "lifecycle-toolkit"
+   update-main:
+     description: "Update the main version of the documentation"
+     required: true
+     default: "false"
+   target-branch:
+     description: "Target branch for the documentation"
+     default: "development"
```

let's please use next as the next version
```suggestion
     default: "next"
```